packages/devtools_app/lib/src/screens/inspector/inspector_tree_controller.dart

```diff
@@ -118,14 +118,14 @@ class InspectorTreeController extends Object
     final firstClient = _clients.isEmpty;
     _clients.add(value);
     if (firstClient) {
-      config.onClientActiveChange?.call(true);
+      config!.onClientActiveChange?.call(true);
```

Comment:
make config late so it can be null safe

Reply:
There is assert in setter that seems to be important:
```
assert(_config == null);
```

Late fields cannot be asserted for null.
There is an alternative to introduce field `_isConfigInitialized`, but is seems to create comparable pollution.

What do you think?

Reply:
we want to make it late final

Reply:
done